### PR TITLE
Removed Nationality for Applicant1 & Applicant2

### PR DESCRIPTION
**Pull request by rajatkgupta29**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description
Removed Nationality for Applicant 1 & 2
### JIRA link
https://tools.hmcts.net/jira/browse/ADOP-557
**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by pankajhmcts — APPROVED** [review]
@rajatkgupta29 
Looks good, please do one round of through testing for applicant1/2 flow and after
@nraghu32 testing only merge the PR.

cc: @abidfs @gauravtomar-gh

File: src/main/app/case/from-api-format.test.ts [hunk 1]
```diff
@@ -89,8 +89,6 @@ describe('from-api-format', () => {
       dateChildMovedIn: { day: '1', month: '1', year: '2021' },
       applicant1AdditionalNames: [{ id: 'MOCK_ID', firstNames: 'MOCK_FIRST_NAMES', lastNames: 'MOCK_LAST_NAMES' }],
       applicant2AdditionalNames: [{ id: 'MOCK_ID', firstNames: 'MOCK_FIRST_NAMES', lastNames: 'MOCK_LAST_NAMES' }],
-      applicant1AdditionalNationalities: ['MOCK_COUNTRY'],
```
Comment (abidfs) [inline src/main/app/case/from-api-format.test.ts:92]: can be reused for birthMother and birthFather additional nationalities

File: src/main/app/case/to-api-format.test.ts [hunk 2]
```diff
@@ -18,8 +18,6 @@ describe('to-api-format', () => {
     applicant1AdditionalNames: [{ id: 'MOCK_ID', firstNames: 'MOCK_FIRST_NAMES', lastNames: 'MOCK_LAST_NAMES' }],
     applicant2HasOtherNames: YesOrNo.YES,
     applicant2AdditionalNames: [{ id: 'MOCK_ID', firstNames: 'MOCK_FIRST_NAMES', lastNames: 'MOCK_LAST_NAMES' }],
-    applicant1AdditionalNationalities: ['MOCK_COUNTRY'],
```
Comment (abidfs) [inline src/main/app/case/to-api-format.test.ts:21]: can be reused for birthMother and birthFather additional nationalities

**Review by abidfs — COMMENTED** [review]
looks good just few comments in unit tests

**Pull request merged**